src/resqml2_0_1/RockFluidOrganizationInterpretation.h

```diff
@@ -0,0 +1,93 @@
+/*-----------------------------------------------------------------------
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"; you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-----------------------------------------------------------------------*/
+#pragma once
+
+#include "resqml2/AbstractGridRepresentation.h"
+#include "resqml2_0_1/AbstractOrganizationInterpretation.h"
+#include "resqml2_0_1/EarthModelInterpretation.h"
+#include "resqml2_0_1/OrganizationFeature.h"
+
+namespace RESQML2_0_1_NS
+{
+	/**
+	* This class is a container for other organizations that are consistent to each others.
+	*/
+	class DLL_IMPORT_OR_EXPORT RockFluidOrganizationInterpretation : public AbstractOrganizationInterpretation
```

Comment:
You probably miss a pushBackRockFluidUnit in this class to put units in your organization. It can be definitely part of another PR.

Reply:
I agree with "Maybe we first make the simple case work :)".
So let's not think you can add units in your organization as a first step (which is strange from a business point of view)